Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM mcr.microsoft.com/dotnet/sdk:7.0.202-alpine3.16 AS build
-WORKDIR Authentication/
+WORKDIR AuthenticationApp/
```

Comment:
Is there any specific reason to rename the existing?

Reply:
ok, i just need to double check the deploy in azure devops that this folder change will not impact anything existing